playbooks/common/openshift-loadbalancer/service.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Populate g_service_nodes host group if needed
+  hosts: localhost
+  connection: local
+  become: no
+  gather_facts: no
+  tasks:
+  - fail: msg="new_cluster_state is required to be injected in this playbook"
+    when: new_cluster_state is not defined
```

Comment:
Are these service plays necessary? I was just following form from the other openshift-* playbooks.